models/sport.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function(sequelize, DataTypes) {
+  var Sport = sequelize.define("Sport", {
+    name: DataTypes.STRING,
+    description: DataTypes.STRING,
+    skill_level: DataTypes.STRING
```

Comment:
Take 'skill_level' out of the Sport model. Skill level should be part of a user_sport table that can indicates an individual users skill in each sport.